<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>myPowWindowUi</class>
 <widget class="QWidget" name="myPowWindowUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>323</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>myPowTest</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>271</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTextEdit" name="textEditNum1"/>
    </item>
    <item>
     <widget class="QTextEdit" name="textEditNum2"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButtonPow">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>200</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>myPow(double,int)</string>
   </property>
  </widget>
  <widget class="QLabel" name="Answer">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>140</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
